sdk/src/test/scala/com/horizen/block/SidechainBlockHeaderTest.scala

```diff
@@ -4,17 +4,34 @@ import java.io.{BufferedReader, BufferedWriter, FileReader, FileWriter}
 
 import com.horizen.fixtures.{CompanionsFixture, ForgerBoxGenerationMetadata, SidechainBlockFixture}
 import com.horizen.params.{MainNetParams, NetworkParams}
+import com.horizen.proposition.VrfPublicKey
+import com.horizen.secret.VrfSecretKey
 import com.horizen.utils.BytesUtils
 import com.horizen.validation.InvalidSidechainBlockHeaderException
-import org.junit.Assert.{assertArrayEquals, assertEquals, assertFalse, assertTrue, fail => jFail}
+import com.horizen.vrf.VrfGeneratedDataProvider
+import org.junit.Assert.{assertArrayEquals, assertEquals, assertTrue, fail => jFail}
 import org.junit.Test
 import org.scalatest.junit.JUnitSuite
 
 import scala.util.{Failure, Success}
 
 class SidechainBlockHeaderTest extends JUnitSuite with CompanionsFixture with SidechainBlockFixture {
+  val vrfGenerationDataSeed = 178
+  //set to true for update vrf related data
+  if (false) {
+    VrfGeneratedDataProvider.updateVrfProof(vrfGenerationDataSeed)
```

Comment:
Can we add to the generated files names some suffix to describe where this fixed proof used, like "blockheader_vrfproof178"? What do you think about this?